doc/2/guides/Metadata/index.md

```diff
@@ -0,0 +1,102 @@
+---
```

Comment:
This repertory name should be `metadata`

Reply:
done